api_project/utils.py

```diff
@@ -0,0 +1,25 @@
+import logging
+
+log = logging.getLogger(__name__)
+
+@staticmethod
+def format_kiinteistotunnus(kiinteistotunnus):
+    if len(kiinteistotunnus) == 17 and kiinteistotunnus.count("-") == 3:
```

Comment:
saattaisin pistää jotkut VAKIO_MUUTTUJAt näille numeroille jotka dokumentoi mitä nää numerot signifikoi. Näin asiaa katsellen tietämättä ominaisuuden kontekstia nää numerot on taikanumeroita

Reviewiä eteenpäin katsonneena tää ei ollu uutta koodia vaan siirsit vaan funkkarin, ymmärrän jos siullakaan ei tähän kontekstia ole että mitä ihmettä nää numerot on